src/pybind/mgr/dashboard/services/proto/gateway.proto

```diff
@@ -10,37 +10,17 @@
 
```

Comment:
yup, we thought about it initially. Atleast for generating the pb2 files we might go with a build time generator rather than versioning it ourselves. But as for the proto files, if we decide to download it during build time, i don't know if all the machine has internet access on their build machines and I think that's one of the reason why we had to put it here.